docs/Reference/Pantheon-API-Objects.md

```diff
@@ -65,6 +65,27 @@ Returned by [eth_getFilterChanges](Pantheon-API-Methods.md#eth_getfilterchanges)
 | **data**             | Data                              | Non-indexed arguments of log.                                                                                                                                                                                       |
 | **topics**           | Array of Data, 32&nbsp;bytes each | [Event signature hash](../Using-Pantheon/Events-and-Logs.md#event-signature-hash) and 0 to 3 [indexed log arguments](../Using-Pantheon/Events-and-Logs.md#event-parameters).  |
 
+## Private Transaction Object
+
+Returned by [eea_getPrivateTransaction](Pantheon-API-Methods.md#eea_getprivatetransaction).
+
+| Key                  | Type                              | Value                                                                           |
+|----------------------|-:-:-------------------------------|---------------------------------------------------------------------------------|
+| **from**             | Data, 20&nbsp;bytes               | Address of the sender.                                                          |
+| **gas**              | Quantity                          | Gas provided by the sender.                                                     |
+| **gasPrice**         | Quantity                          | Gas price provided by the sender in Wei.                                        |
+| **hash**             | Data, 32&nbsp;bytes               | Hash of the transaction.                                                        |
+| **input**            | Data                              | Data to create or invoke contract.                                                                                 |
+| **nonce**            | Quantity                          | Number of transactions made by the sender to the privacy group before this one.                      |
+| **to**               | Data, 20&nbsp;bytes               | `null` if a contract creation transaction; otherwise, contract address             |
```

Comment:
a few missing fullstops in the table

Reply:
That's consistent with the rest of the tables. We can revisit this on the wider basis when we look at the style guide